controllers/actions.github.com/resourcebuilder.go

```diff
@@ -741,3 +737,15 @@ func trimLabelValue(val string) string {
 	}
 	return val
 }
+
+func mergeLabels(labels ...map[string]string) map[string]string {
```

Comment:
Can we do something like:
```
func mergeLabels(base, overwrites map[string]string) map[string]string
```
It would be much easier to understand by reading the function definition.